src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -111,10 +111,39 @@ class TestRequestRateManager : public TestLoadManagerBase,
             params.batch_size, params.measurement_window_ms, params.max_threads,
             params.num_of_sequences, params.sequence_length,
             params.shared_memory_type, params.output_shm_size,
-            params.start_sequence_id, params.sequence_id_range,
-            params.string_length, params.string_data, params.zero_input,
-            params.user_data, GetParser(), GetFactory())
+            params.start_sequence_id, params.sequence_id_range, GetParser(),
+            GetFactory())
   {
+    InitManager(
+        params.string_length, params.string_data, params.zero_input,
+        params.user_data);
+  }
+
+  TestRequestRateManager(
```

Comment:
Can you add some documentation to this function since it's quite a specific use-case?

Reply:
done